core/src/test/java/nl/nn/adapterframework/jdbc/dbms/MariaDbDbmsSupportTest.java

```diff
@@ -1,23 +1,23 @@
 package nl.nn.adapterframework.jdbc.dbms;
 
-import static org.junit.Assert.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 
-public class MariaDbDbmsSupportTest {
+class MariaDbDbmsSupportTest {
 
 	@ParameterizedTest
 	@CsvSource({"9.9", "5.5.5-10.4.26-MariaDB-log"})
-	public void testHasNoSkipLocked(String version) {
+	void testHasNoSkipLocked(String version) {
```

Comment:
Ik weet dat @nielsm5 er erg gesteld op is om overal voor alle methods een access-modifier te zien, dus ook voor alle JUnit 5 test methods (ook al is het daar niet nodig).

Reply:
Kan me dat bij Niels niet voorstellen.. maar anders komt hij er vast zelf mee ;-)

Reply:
Haha, ik ben zelf erg van het expliciete ipv alles impliciet te maken. Misschien een autistisch trekje..?
Sinds Java 21 begint het steeds meer te lijken op JavaScript terwijl JavaScript er alles aan doet om TypeScript te promoten. 
Alles nu var gaan noemen, static's die niet meer nodig zijn en alle access modefiers weghalen is wat mij betreft geen verbetering tov hoe het altijd geweest is, maar het wordt nu wel 'Beginner-Friendly Java Programming' genoemd. 

Ik weet dat het niet meer nodig is voor JUnit 5 maar los van dat ik het onzin vind om nu overal public weg te halen (omdat het kan/niet meer nodig is), vind ik het ook zonde van de tijd/energie vind om alles nu aan te gaan passen. De codebase is hier te groot voor en mijns inziens geeft het (met modifiers) meer inzicht wat het moet doen.

Reply:
Ben het met je eens dat we geen tijd moeten besteden aan de publics verwijderen bij JUnit tests. Qua programmeermodel is het niet verkeerd, aangezien de unit tests niet vanuit andere packages aangeroepen hoeven te worden. Dus ik ben het er inhoudelijk wel mee eens. 

Vanaf Java 10 kun je al `var` gebruiken, maar ik ben daar niet per se een voorstander van... enfin, dat is ook een beetje een andere discussie buiten de scope van deze PR ;-)